src/bzflag/DisplayMenu.cxx

```diff
@@ -442,8 +473,8 @@ void            DisplayMenu::callback(HUDuiControl* w, const void* data)
     }
     case 'A':
```

Comment:
you removed several options but their callback code remains (e.g., '1' : dithering). Only the cases that can be executed should remain.

Reply:
No display options were removed.  They were just re-ordered, and two of the debugging options were merged into a single item.